go/src/koding/klient/machine/transport/rsync/rsync.go

```diff
@@ -17,6 +18,13 @@ import (
 	"koding/klient/machine/index"
 )
 
+// englishEnv contains current environment with C locale.
+var englishEnv []string
+
+func init() {
+	englishEnv = append(os.Environ(), "LC_ALL=C")
```

Comment:
Also `LANG` and use `(koding/os.Environ).Encode`, if os.Environ() has LC_ALL already defined, this append won't overwrite it.